app/helpers/rsvps_helper.rb

```diff
@@ -1,2 +1,7 @@
 module RsvpsHelper
+  def markdown(text)
+    renderer = Redcarpet::Render::HTML.new(filter_html: true, no_images: true, no_styles: true, hard_wrap: true)
+    markdown = Redcarpet::Markdown.new(renderer)
```

Comment:
Did a quick benchmark, these objects are very cheap to create, so leaving it as-is.